packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -775,6 +787,9 @@ export class DbAuthHandler {
 
       return newUser
     }
+
```

Comment:
Wanted to comment on a line a few lines up, but GH wouldn't let me, so I post it here instead

```diff
- const newUser = await this.options.signup.handler({
+ const newUser = await this.options.signup.handler<TUser>({
```

And again, like 50 lines below this, I wanted to suggest this change
```diff
- _loginResponse(user: Record<string, any>, statusCode = 200) {
+ _loginResponse(user: Record<string, unknown>, statusCode = 200) {
```